scripts/setup-ubuntu.sh

```diff
@@ -140,7 +140,7 @@ function install_conda {
 
   mkdir -p conda && cd conda
   wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-$ARCH.sh
-  bash Miniconda3-latest-Linux-$ARCH.sh -b -p $MINICONDA_PATH
+  ${SUDO} bash Miniconda3-latest-Linux-$ARCH.sh -b -p $MINICONDA_PATH
```

Comment:
I think it makes more sense to make the path configurable: https://github.com/facebookincubator/velox/pull/9789/files#diff-960a11a4bac38369ab5702ab18b60cac5dff8a4e14a732b4adfc75964bc23b22L130 as it's explicitly a feature of miniconda NOT to need sudo: https://docs.anaconda.com/free/miniconda/miniconda-install/

Reply:
Thanks for the advice. Update.